examples/tutorial/x/blog/errors.go

```diff
@@ -5,6 +5,6 @@ var (
 	invalidText               = "text is too long or too short"
 	invalidName               = "name is too long"
 	descriptionTooLong        = "description is too long"
-	unauthorisedBlogAuthorFmt = "blog author %X"
-	unauthorisedPostAuthorFmt = "post author %X"
+	unauthorisedBlogAuthorFmt = "blog author %#v"
```

Comment:
Why `%#v`, not `%s` ?

Reply:
because there's nil passed as well in some cases. %#v supports both cases

Reply:
Good point